Firebase/Core/FIRApp.m

```diff
@@ -131,6 +131,21 @@ + (void)configureDefaultAppWithOptions:(FIROptions *)options
       [FIRApp sendNotificationsToSDKs:sDefaultApp];
       sDefaultApp.alreadySentConfigureNotification = YES;
     }
+
+#if DEBUG
```

Comment:
Why #if DEBUG?

Presumably this won't get built into release builds.

Should we test for not from app store instead?

Reply:
I figured since it's on a critical path, it should be removed entirely. Is there a reason this wouldn't get into release builds without it?

Reply:
We're not releasing source pods yet - so there is only one FirebaseCore release and DEBUG is not defined in it.

This change would be fine once we have source pod releases.

Reply:
Release builds don't have the `DEBUG` flag defined (since we ship binaries).

Reply:
*facepalm* right - thanks. I'll change to App Store check.